synapse/tests/test_tools_backup.py

```diff
@@ -47,35 +47,40 @@ def compare_dirs(self, dir1, dir2, skipfns=None, skipdirs=None):
 
     async def test_backup(self):
 
-        async with self.getTestCore() as core:
-            layriden = core.getLayer().iden
+        with self.getTestDir() as dirn:
+            async with self.getTestCore(dirn=dirn) as core:
+                layriden = core.getLayer().iden
 
-            # For additional complication, open a spooled set that shouldn't be backed up
-            async with await s_spooled.Set.anit(dirn=core.dirn, size=2) as sset:
-                await sset.add(10)
-                await sset.add(20)
-                await sset.add(30)
+                # For additional complication, open a spooled set that shouldn't be backed up
+                async with await s_spooled.Set.anit(dirn=core.dirn, size=2) as sset:
```

Comment:
leaving the context here will remove the spool set slab so the fact its skipped in the backup is meaningless?

Reply:
That's a good point.

Reply:
fixed